SimCalorimetry/HGCalSimProducers/src/HGCDigitizer.cc

```diff
@@ -561,7 +837,15 @@ void HGCDigitizer::resetSimHitDataAccumulator() {
     it->second.hit_info[1].fill(0.);
   }
 }
-
+void HGCDigitizer::resetPUSimHitDataAccumulator() {
+  for (HGCPUSimHitDataAccumulator::iterator it = pusimHitAccumulator_->begin(); it != pusimHitAccumulator_->end();
```

Comment:
range-based loop?

Reply:
actually, this function doesn't appear to be used anywhere, it could just be deleted